hr_expense_substate/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,5 @@
+This module allows to add a substate to an expense report.
+For each expense report state you can define a substate.
+We this module you can define substate which allow you to extend expense report workflow.
```

Comment:
I think this line can be deleted. The improved one is already above

Reply:
This description for explain this module. I think it okay to use this.

Reply:
`We this module` is not proper english :)
The line says the same as the lines above, so it is a duplicate.

Reply:
@Saran440 Going to review this one now. Can you please attend to my earlier comment.